app/src/main/java/br/com/alexf/boraprofut/features/game_screen/usecase/GameScreenUseCase.kt

```diff
@@ -0,0 +1,21 @@
+package br.com.alexf.boraprofut.features.game_screen.usecase
+
+import android.util.Log
+import br.com.alexf.boraprofut.features.game_screen.mock.teamList
+import br.com.alexf.boraprofut.features.game_screen.model.ReadyMadeGames
+import kotlin.text.Typography.times
+
+class GameScreenUseCase {
+    fun getGames() : List<ReadyMadeGames> {
+        val timesEmbaralhados = teamList.shuffled()
+        val listaPartidas = mutableListOf<ReadyMadeGames>()
+        if (timesEmbaralhados.size % 2 == 0) {
```

Comment:
o problema desse código é que se for falso, ele não faz nada, certo?

Reply:
ajustado